journal/src/main/scala/com/evolutiongaming/kafka/journal/JsValueCodec.scala

```diff
@@ -0,0 +1,49 @@
+package com.evolutiongaming.kafka.journal
+
+import com.evolutiongaming.jsonitertool.PlayJsonJsoniter
+import play.api.libs.json.{JsError, JsResult, JsSuccess, JsValue, Json}
+import scodec.bits.ByteVector
+
+import scala.util.{Failure, Success, Try}
+
+trait JsValueCodec {
```

Comment:
Could you please divide them? Encode and Decode operations are in separate interfaces. 
Could you also go through all usages and use one or another